pkg/ebpf/policies_manager_test.go

```diff
@@ -0,0 +1,81 @@
+package ebpf
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/aquasecurity/tracee/pkg/policy"
+)
+
+func TestPoliciesManagerEnablePolicy(t *testing.T) {
+	policiesManager := &policiesManager{policyMask: 0}
+
+	for i := 0; i < 64; i++ {
```

Comment:
Here you can use `policy.MaxPolicies` instead of 64. We have plans to increase it.